templates/mobile_view_activity_branching_page.mustache

```diff
@@ -0,0 +1,195 @@
+{{!
+    This file is part of Moodle - http://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{=<% %>=}}
+<div>
```

Comment:
Always start templates with an ionic wrapper please.

Reply:
so this needs to move right to the top?

Reply:
Rather replace the div with something like ion-card or ion-item

Reply:
when I remove this tag I get a template error, got it from the templates that were already created by the dev who worked on the original questionnaire mobile

Reply:
Strange should work. Leave for now can look at later, not a show stopper at this point